packages/clerk-js/src/core/clerk.ts

```diff
@@ -1324,6 +1312,24 @@ export default class Clerk implements ClerkInterface {
     });
   };
 
+  #buildSignInOrUpUrl = (key: 'signInUrl' | 'signUpUrl', options?: RedirectOptions): string => {
+    const opts: RedirectOptions = {
+      afterSignInUrl: extractAuthPathByPriority('afterSignInUrl', { ctx: options, options: this.#options }, false),
+      afterSignUpUrl: extractAuthPathByPriority('afterSignUpUrl', { ctx: options, options: this.#options }, false),
```

Comment:
❓ shall we rename `options?` param to `context?` ? I think it's a little confusing that `{ ctx: options, options: this.#options }` we have 2 options here.